### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ENOCKMOSETI**
#### Is your pull request related to a problem? Please describe
A clear and concise description of what the problem is. Ex. I'm always frustrated when ...

#### Test plan
A good test plan should give instructions that someone else can easily follow.
Also add a link to your own Messenger experience for your reviewer to test the changes.

#### Checklist
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [ ] I have added the label **enhancement** to my pull request.

### CODE REVIEW

**Comment by hoduchieu01** [comment]
This pull request fixed issue #11

**Comment by hoduchieu01** [comment]
@ENOCKMOSETI This branch has conflicts so you need to solve these conflicts and after that we can merge this pull request.

**Comment by hoduchieu01** [comment]
@ENOCKMOSETI tackle this issue on PR-CI check https://github.com/fbdevelopercircles/open-source-edu-bot/pull/38/checks

File: src/services/messenger.py [hunk 1]
```diff
@@ -301,8 +301,8 @@ def process_postback(messenger, payload):
         qrs = quick_replies.QuickReplies(quick_replies=[qr])
         text = {
             "text": _(
-                u'😎 Now that you understand what Version control is,'
-                ' let\'s explore another important topic.'
+                u'😎 Now That you understand what version control is,'
```
Comment (elinguiuriel) [inline src/services/messenger.py:304]: why `That` with `T` in uppercase?

File: src/services/messenger.py [hunk 2]
```diff
@@ -247,8 +247,8 @@ def process_postback(messenger, payload):
         qrs = quick_replies.QuickReplies(quick_replies=[qr])
         text = {
             "text": _(
-                u'👩🏽‍🏫 You know ...\n✔️ Wordpress,\n✔️ Notepad++,\n✔️ Ubuntu\n'
-                'and thousand of common software started out as open source'
+                u'👩🏽‍🏫 Did you know ...\n✔️ Wordpress,\n✔️ Notepad++,\n✔️ Ubuntu\n'
```
Comment (elinguiuriel) [inline src/services/messenger.py:250]: This line have more than 80 characters and will not pass the flake8 test, break it in 2!

File: src/services/messenger.py [hunk 3]
```diff
@@ -403,7 +403,7 @@ def process_postback(messenger, payload):
             payload='INSTALL_GIT'
         )
         qr2 = quick_replies.QuickReply(
-            title=_('🤓 I\'ve Git Installed'),
+            title=_('🤓 I\'ve got Git Installed'),
```
Comment (elinguiuriel) [inline src/services/messenger.py:406]: The title should not have more that [20 characters](https://developers.facebook.com/docs/messenger-platform/reference/buttons/quick-replies). Please restore the previous formulation!

**Comment by aboullaite** [comment]
Thank you @ENOCKMOSETI for bearing with us! This PR is close to merge, it just need to have @elinguiuriel remarks and fix conflicts 👍

**Comment by ENOCKMOSETI** [comment]
> Please update your pull request regarding my comments
> Also please make your PR in `develop `branch

Updated pull request regarding comments
opened a PR in develop branch

**Pull request closed without merging**